css/style.css

```diff
@@ -1,32 +1,41 @@
-/**
- * Beware that Qt Assistant (QTextBrowser) supports only a limited set of 
- * HTML/CSS: http://qt-project.org/doc/qt-5/richtext-html-subset.html
- */
```

Comment:
No reason to be pessimistic. It is only a reminder to make the design "degrade gracefully". However, we don't need to be cutting-edge in design as long as the contents lag two years behind the released software.